lib/manageiq/api/open_api/generator.rb

```diff
@@ -20,9 +20,33 @@ def initialize
 
         def generate!
           openapi_spec["components"]["schemas"] = build_schemas
+          openapi_spec["components"]["parameters"] = build_common_parameters
+          openapi_spec["paths"] = build_paths
           openapi_path.write("#{JSON.pretty_generate(openapi_spec)}\n")
         end
 
+        # Helper method to safely get model class from klass (string or class)
+        def get_model_class(klass)
+          case klass
+          when String
+            begin
+              klass.constantize
+            rescue NameError => e
+              puts "Warning: Could not constantize #{klass}: #{e.message}"
+              nil
+            end
+          when Class
+            klass
+          else
+            nil
+          end
+        end
```

Comment:
This looks like it could be just `klass = klass.safe_constantize if klass.kind_of?(String)` 
If we don't need to handle `Class or String` and only `String` then we could use safe_constantize directly.